.circleci/config.yml

```diff
@@ -92,34 +92,44 @@ jobs:
       xcode: "9.0"
     steps:
       - checkout
-      - run: make dependencies
-      - run: make test
+      - run: brew update
```

Comment:
Not keen on this but without updating the pyenv formula we can't install python 3.5.8